pkg/controller/controller.go

```diff
@@ -630,6 +632,67 @@ func (cont *AciController) Run(stopCh <-chan struct{}) {
 			cont.opflexDeviceDeleted(dn)
 		})
 	go cont.apicConn.Run(stopCh)
+	go cont.enableGlobalSync(stopCh)
+}
+
+func (cont *AciController) enableGlobalSync(stopCh <-chan struct{}) {
+	time.Sleep(time.Minute)
+	for {
+		cont.log.Debug("Running code to sync GlobalInfo with Node infos")
```

Comment:
"Running code to sync" -> "Syncing "

and we should consider logging this every perhaps every 10th iteration of the loop or something to reduce the number of logs